dev/com.ibm.ws.jca_fat_regr/fat/src/suite/r80/base/jca16/TestSetupUtils.java

```diff
@@ -0,0 +1,220 @@
+/*******************************************************************************
+ * Copyright (c) 2022 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package suite.r80.base.jca16;
+
+import java.io.File;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+import org.jboss.shrinkwrap.api.ArchivePath;
+import org.jboss.shrinkwrap.api.Filter;
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.spec.EnterpriseArchive;
+import org.jboss.shrinkwrap.api.spec.JavaArchive;
+import org.jboss.shrinkwrap.api.spec.ResourceAdapterArchive;
+import org.jboss.shrinkwrap.api.spec.WebArchive;
+
+import com.ibm.websphere.simplicity.ShrinkHelper;
+
+import componenttest.topology.impl.LibertyServer;
+
+/**
+ * Class <CODE>TestSetupUtils</CODE> provides utility methods used
+ * to help set up resources needed for tests
+ */
+public class TestSetupUtils {
+
+    private final static String CLASSNAME = TestSetupUtils.class.getName();
+    private final static Logger LOGGER = Logger.getLogger(CLASSNAME);
```

Comment:
Instead of registering a logger, we should be using the simplicity logger. 
`com.ibm.websphere.simplicity.log.Log`